### PR TITLE
fixed formatting of who text box, including collapse and expand arrow

### PR DESCRIPTION
**Pull request by maiseyperelonia**
see video https://utoronto-my.sharepoint.com/:v:/g/personal/maisey_perelonia_mail_utoronto_ca/EZnkzzQQMUlEmoQhXcDHdDUBBKvPP71rg986NHITw4S4KQ?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJPbmVEcml2ZUZvckJ1c2luZXNzIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXciLCJyZWZlcnJhbFZpZXciOiJNeUZpbGVzTGlua0NvcHkifX0&e=wrxZSQ

### CODE REVIEW

File: prod_website/index.html [hunk 1]
```diff
@@ -23,15 +23,30 @@
...
+    </script>
+
   </head>
 
   <body>
     <div class="se-pre-con"></div>
-    <header id="home">
-      <nav style="background-color: #fff; padding: 10px; display: flex; justify-content: space-between; align-items: center; width: 100%; box-sizing: border-box;">
+    <header id="home" style="margin-bottom: 0px;">
+      <nav style="background-color: #B3CEE5; padding: 10px; display: flex; justify-content: space-between; align-items: center; width: 100%; box-sizing: border-box;">
```
Comment (hopehadfield) [inline prod_website/index.html:46]: It looks good, but I think we should make the background black so it's seamless with the rest of the header. Thoughts @katzyx @floriafz23 ?

Comment (katzyx) [reply prod_website/index.html:46]: i agree

**Comment by maiseyperelonia** [comment]
gonna fix accounts later

**Review by katzyx — APPROVED** [review]
lgtm

**Pull request merged**